pages/_app.tsx

```diff
@@ -1,23 +1,25 @@
 import { useEffect, useState, createContext } from "react";
-import "@/styles/globals.scss";
-import colors from "@/utils/colors";
-import Header from "@/components/Header";
+import "../styles/globals.scss";
+import colors from "../utils/colors";
+import Header from "../components/Header";
+import type { AppProps } from 'next/app'
 
-export const ThemeContext = createContext(null);
+type Themes= "bright" | "dark";
+export const ThemeContext = createContext({});
 
-export default function App({ Component, pageProps }) {
-	const [theme, setTheme] = useState("bright");
+export default function App({ Component, pageProps }: AppProps) {
+	const [theme, setTheme] = useState<Themes>("bright");
 
 	useEffect(() => {
 		if (!localStorage.getItem("fontSize")) {
-			localStorage.setItem("fontSize", 16);
+			localStorage.setItem("fontSize", "16");
```

Comment:
there isn't really a need for this change - everything you put in the 2nd parameter is serialized, so is the 16